pkgdb/include/flox/lock-flake-installable.hh

```diff
@@ -77,6 +77,7 @@ struct LockedInstallable
   std::optional<std::vector<std::string>> licenses;
   std::optional<bool>                     broken;
   std::optional<bool>                     unfree;
+  std::optional<int64_t>                  priority;
```

Comment:
nonblocking: don't think it will matter, but why isn't this a NixInt or a uint64_t?

Reply:
Just a typo